api/tests/unit/app_analytics/test_unit_app_analytics_views.py

```diff
@@ -131,3 +145,73 @@ def test_get_total_usage_count_for_non_admin_user_returns_403(
 
     # Then
     assert response.status_code == status.HTTP_403_FORBIDDEN
+
+
+@pytest.mark.skipif(
+    "analytics" not in settings.DATABASES,
+    reason="Skip test if analytics DB is not configured",
+)
+@pytest.mark.django_db(databases=["default", "analytics"])
+def test_set_sdk_analytics_flags_with_identifier(
+    api_client: APIClient,
+    environment: Environment,
+    feature: Feature,
+    identity: Identity,
+    settings: SettingsWrapper,
+) -> None:
+    # Given
+    settings.USE_POSTGRES_FOR_ANALYTICS = True
+    url = reverse("api-v1:analytics-flags")
+    url += f"?identity_identifier={identity.identifier}"
+    api_client.credentials(HTTP_X_ENVIRONMENT_KEY=environment.api_key)
+    feature_request_count = 2
+    data = {feature.name: feature_request_count}
+
+    # When
+    response = api_client.post(
+        url, data=json.dumps(data), content_type="application/json"
+    )
+
+    # Then
+    assert response.status_code == status.HTTP_200_OK
+
+    assert FeatureEvaluationRaw.objects.count() == 1
+    feature_evaluation_raw = FeatureEvaluationRaw.objects.first()
+    assert feature_evaluation_raw.identity_identifier == identity.identifier
+    assert feature_evaluation_raw.feature_name == feature.name
+    assert feature_evaluation_raw.environment_id == environment.id
+    assert feature_evaluation_raw.evaluation_count == feature_request_count
+
+
+@pytest.mark.skipif(
+    "analytics" not in settings.DATABASES,
+    reason="Skip test if analytics DB is not configured",
+)
+@pytest.mark.django_db(databases=["default", "analytics"])
+def test_set_sdk_analytics_flags_with_enabled_when_evaluated(
+    api_client: APIClient,
+    environment: Environment,
+    feature: Feature,
+    settings: SettingsWrapper,
+) -> None:
+    # Given
+    settings.USE_POSTGRES_FOR_ANALYTICS = True
+    url = reverse("api-v1:analytics-flags")
+    url += "?enabled_when_evaluated=true"
```

Comment:
So I guess this means that _all_ features sent in the request will be marked as 'enabled_when_evaluated` ? This might make the integration pretty tricky in all SDKs since they all keep a local cache (with counts) and then flush that periodically. It's possibly ok for now, but it's going to require some heavy changes in any of the SDKs that we want to implement this in. 

It's making me think that we could just create a brand new endpoint...

Reply:
Ok this is now implemented.